src/os/filestore/FileStore.h

```diff
@@ -48,10 +48,8 @@
 
 #include "include/uuid.h"
 
-
-// from include/linux/falloc.h:
-#ifndef FALLOC_FL_PUNCH_HOLE
-# define FALLOC_FL_PUNCH_HOLE 0x2
+#ifdef __linux__
+#include <linux/falloc.h>
```

Comment:
could you move this #include into `FileStore.cc`?

Reply:
@tchaikov yes, I've done it.